README.md

```diff
@@ -2,43 +2,40 @@
 
 ## Prerequisites
 
-* `nodejs` en `yarn`
+* `nodejs`, `npm`
 
-Use `npm` to install `yarn`:
+## Getting started
 
 ```
-> npm install yarn
+>  Use npm install
```

Comment:
`install yarn`. no "use", 'use' is unneeded.. One could argue that `npm install` is also unneeded..